assets/list_assets.py

```diff
@@ -0,0 +1,142 @@
+import sublime_plugin
+import sublime
+import mdpopups
+import re
+
+from pieces_os_client import *
+from pieces import config
+
+from . import assets_identifiers_ws
+
+max_assets = 10
```

Comment:
this should be either a class property or a static class property instead of a global variable